src/stories/Foundation/Table/TableRow.tsx

```diff
@@ -75,10 +78,25 @@ export const TableRow = ({
     setIsEditing(false);
   };
 
+  const onClick = () => {
+    if (!onClickRow) return;
+
+    onClickRow(row.id);
+  };
+
+  // stop click event to prevent the click event on the row to be called
+  const stopClickEvent = (e: React.MouseEvent) => {
+    e.stopPropagation();
+  };
+
   return (
-    <TR selected={selectedRows && selectedRows.includes(row.id)}>
+    <TR
+      selected={selectedRows && selectedRows.includes(row.id)}
+      onClick={onClick}
+      clickable={!!onClickRow}
```

Comment:
This is nitpicking, but I feel like we have prepended a lot of state props with is, change to isClickable?